hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/WorkflowsToDeleteMetadataMetricHelperActor.scala

```diff
@@ -0,0 +1,64 @@
+package cromwell.services.metadata.hybridcarbonite
+
+import java.time.OffsetDateTime
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
+import cats.data.NonEmptyList
+import cromwell.core.instrumentation.InstrumentationPrefixes
+import cromwell.services.MetadataServicesStore
+import cromwell.services.instrumentation.CromwellInstrumentation
+import cromwell.services.metadata.MetadataArchiveStatus
+import cromwell.services.metadata.MetadataArchiveStatus.Archived
+import cromwell.services.metadata.hybridcarbonite.DeleteMetadataActor.MetricActorFreed
+import cromwell.services.metadata.hybridcarbonite.WorkflowsToDeleteMetadataMetricHelperActor._
+import cromwell.services.metadata.impl.{MetadataDatabaseAccess, MetadataServiceActor}
+
+import scala.util.{Failure, Success}
+
+class WorkflowsToDeleteMetadataMetricHelperActor(metadataDeletionActor: ActorRef, override val serviceRegistryActor: ActorRef)
+  extends Actor
+  with ActorLogging
+  with MetadataDatabaseAccess
+  with MetadataServicesStore
+  with CromwellInstrumentation {
+
+  implicit val ec = context.dispatcher
+
+  override def postRestart(reason: Throwable): Unit = {
+    super.postRestart(reason)
+    metadataDeletionActor ! MetricActorFreed
+  }
+
+  override def receive: Receive = {
+    case SendPrecalculatedMetric(workflowsToDeleteMetadata) =>
```

Comment:
I would give these messages more specific names - it took me a while to realize that they were specific messages for "archived workflows deletion queue" metrics